packages/pangomm_1_4.rb

```diff
@@ -26,5 +26,7 @@ class Pangomm_1_4 < Meson
   depends_on 'mm_common' => :build
```

Comment:
Update to [2.52.0](https://gitlab.gnome.org/GNOME/pangomm/-/tags/2.52.0)

Reply:
I think that would be pango_2_48...